docs/conf.py

```diff
@@ -22,20 +23,74 @@
 import tmt.utils
 from tmt.utils import Path
 
-try:
-    # use bootstrap theme if user has it installed
-    import sphinx_bootstrap_theme
-    HTML_THEME = 'bootstrap'
-    html_theme_path = [sphinx_bootstrap_theme.get_html_theme_path()]
-except ImportError:
+_POSSIBLE_THEMES: List[Tuple[Optional[str], str]] = [
+    # Fall back to sphinx_rtd_theme if available
+    ('sphinx_rtd_theme', 'sphinx_rtd_theme'),
+    # The default theme
+    (None, 'default')
+    ]
+
+# NOTE: this one is defined somewhere below, among original Sphinx config fields,
+# but we need it as early as possible to be set when loading themes.
+# Add any paths that contain custom themes here, relative to this directory.
+html_theme_path = []
+
+
+def _load_theme(
+        theme_package_name: str,
+        theme_name: str) -> bool:
     try:
-        # fall back to sphinx_rtd_theme if available
-        import sphinx_rtd_theme
-        HTML_THEME = 'sphinx_rtd_theme'
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-    except ImportError:
-        # and fall back to 'default' if neither of those are available
-        HTML_THEME = 'default'
+        theme_package = importlib.import_module(theme_package_name)
+
+    except ModuleNotFoundError:
+        return False
+
+    global HTML_THEME, html_theme_path
+
+    if hasattr(theme_package, 'get_html_theme_path'):
+        global html_theme_path
+
+        HTML_THEME = theme_name
+
+        path = theme_package.get_html_theme_path()
+
+        html_theme_path = path if isinstance(path, list) else [path]
+
+        return True
+
+    HTML_THEME = theme_name
+
+    return True
+
+
+if 'TMT_DOCS_THEME' in os.environ:
```

Comment:
Would be nice to document this varialble.

Reply:
Added in 88497cc9.

Reply:
Thanks! Nice!